src/mlio/record_readers/blob_record_reader.cxx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You
+ * may not use this file except in compliance with the License. A copy of
+ * the License is located at
+ *
+ *      http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is
+ * distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+ * ANY KIND, either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ */
+#include "mlio/record_readers/blob_record_reader.h"
+
+#include <mlio/logger.h>
+#include "mlio/memory/memory_slice.h"
+#include "mlio/record_readers/record.h"
+#include "mlio/span.h"
```

Comment:
Looks like you don't need to include `span.h` here. We do not use it anywhere in the implementation.